src/main/java/seedu/address/model/person/Person.java

```diff
@@ -111,13 +102,14 @@ public boolean equals(Object other) {
                 && otherPerson.getPhone().equals(getPhone())
                 && otherPerson.getEmail().equals(getEmail())
                 && otherPerson.getAddress().equals(getAddress())
-                && otherPerson.getSkills().equals(getSkills());
+                && otherPerson.getSkills().equals(getSkills())
+                && otherPerson.getModules().equals(getModules());
```

Comment:
I was a bit worried about this but after reading JavaDoc for Set interface, it says:

>Returns true if the specified object is also a set, the two sets have the same size, and every member of the specified set is contained in this set (or equivalently, every member of this set is contained in the specified set).

So it compares every Skill and Module in respective sets, and since their `equals` method have been well implemented, this should work!

Reply:
I'll add some tests in the future to confirm this.